src/modules/Dashboard/components/ActivityAssignDrawer/ActivitiesList/ActivitiesList.tsx

```diff
@@ -26,6 +27,8 @@ export const ActivitiesList = ({
   'data-testid': dataTestId,
 }: ActivitiesListProps) => {
   const { t } = useTranslation('app');
+  const { featureFlags } = useFeatureFlags();
+  const enableActivityAssignFlag = featureFlags.enableActivityAssign;
```

Comment:
I don't think the feature flag check is needed since the parent component that uses this (`ActivityAssignDrawer`) is effectively behind the `enableActivityAssign` feature flag wherever it's used.